src/seedu/addressbook/AddressBook.java

```diff
@@ -54,7 +54,7 @@
     /**
      * A platform independent line separator.
      */
-    private static final String LS = System.lineSeparator() + LINE_PREFIX;
+    private static final String lineSeparator = System.lineSeparator() + LINE_PREFIX;
```

Comment:
This is a good catch.

I encourage you to try find some more coding standard violations in the given code base.